selectrum.el

```diff
@@ -1082,7 +1084,12 @@ is very confusing.
 
 MAY-MODIFY-CANDIDATES, if non-nil, means that Selectrum is
 allowed to modify the CANDIDATES list destructively. Otherwise a
-copy is made."
+copy is made.
+
+For MINIBUFFER-COMPLETION-TABLE and
+MINIBUFFER-COMPLETION-PREDICATE see `minibuffer-completion-table'
+and `minibuffer-completion-predicate'. They are used for internal
+purposes and compatibility to Emacs completion API."
```

Comment:
This should probably explicitly mention that passing the keyword arguments will dynamically bind the corresponding variables, as otherwise you have to be extremely familiar with binding semantics and the implementation of `cl-defun` to guess that it has this effect.

Reply:
Good idea, have done it.